Correct FK column name errors in SQL Script

stewartl97 opened the pull request:
Corrected remaining FK creation issues that remained after converting ID to UUID.

## 🗣 Description ##

Modify column. names in SQL Foreign Key creation to proper names after columns were updated to support UUID's.

## 💭 Motivation and context ##

This change allows the SQL script to be run without manual intervention to correct for errors.

## 🧪 Testing ##

Ran the script against my test database and verified tables were created and keys were linked as expected.

Run on PostgreSQL database server through the Query tool.

## ✅ Checklist ##

- [x] Script can be run as a single click with no intervention to create schema.

stewartl97 committed:
Modify to make SQL script compilable

Corrected remaining FK creation issues that remained after converting to UUID.

schmelz21 reviewed (approved):
DB loads clean with correction.

cduhn17 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -60,7 +60,7 @@ CREATE TABLE public.executives
 (
     dnstwist_uid uuid default uuid_generate_v1() NOT NULL,
-    "discoveredBy" text NOT NULL,
+    "discoveredBy" uuid NOT NULL,
```
```suggestion
    "discoveredBy" text NOT NULL,
```
There is already a UUID column. Is it the intention to have a second UUID?

stewartl97 replied:
This resolves the Postgresql error concerning keys with different data types.  The UUID column referenced here is a foreign key with it's parent being the domains table column domain_uid.

cduhn17 replied:
Ok that makes sense. I understand the relationship.

schmelz21 committed:
Update data_schema.sql

Second glance - Lets remove this too.

cduhn17 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -40,7 +33,7 @@ CREATE TABLE IF NOT EXISTS public.domains
 CREATE TABLE public.alias
```
```suggestion
CREATE TABLE IF NOT EXISTS public.alias
```
Add for standard on table creation

cduhn17 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -49,7 +42,7 @@ CREATE TABLE public.alias
 CREATE TABLE public.executives
```
```suggestion
CREATE TABLE IF NOT EXISTS public.executives
```

DJensen94 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -112,15 +105,15 @@ CREATE TABLE public.mentions
 CREATE TABLE IF NOT EXISTS public.hibp_breaches
 (
     hibp_breaches_uid uuid default uuid_generate_v1() NOT NULL,
-    breach_name text NOT NULL,
```
We need to keep the breach_name, it doesn't have to be the primary key, but we want to keep the name

cduhn17 replied:
@DJensen94 
Per commit [08a378b](https://github.com/cisagov/pe-reports/pull/60/commits/08a378b5de8b2771c19647c49b8ff5e65ba6d9d4) the column name as been inserted as seen in your last comment.

cduhn17 committed:
Add "IF NOT EXISTS" at table create

cduhn17 committed:
Add "IF NOT EXISTS" at table create

cduhn17 committed:
Add column name and "IF NOT EXISTS" 

The column "breach name" has been added back to CSG credentials table. Also added "IF NOT EXISTS to all remaining create table statements.

DJensen94 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -144,6 +138,7 @@ CREATE TABLE IF NOT EXISTS public.hibp_exposed_credentials
     root_domain text,
```
Line above this org, should be changed to organization_uid, and a FK should be added

DJensen94 commented on src/pe_reports/data/data_schema.sql:
```diff
@@ -153,7 +148,7 @@ CREATE TABLE IF NOT EXISTS public.cybersix_exposed_credentials
 (
```
need to add a organization_uid as a foreign key here as well

DJensen94 committed:
Added Foreign Keys linking organizations to HIBP

Added Foreign keys and relationships to HIBP linking it to Organizations

DJensen94 committed:
Add Shodan Vound vulns tables and foreign key relationships to Schema

Add three Shodan tables to the database including the Assets and IPs Shodan idetnified, the verified vulnerabilities, and the the unverified vulnerabilities/ insecure protocols.

DJensen94 committed:
Add cyhy db name to the organizations table

Added a column to the organizations table for the cyhy database name. This will allow us to link our database to the Crossfeed database as well as have the names necesarry to query the cyhy databse to retrieve encryption data

DJensen94 committed:
Added Credential count to hibp table

this field tells how many total breaches were found in the breach. Will be needed in future iterations of the report

DJensen94 committed:
Updated value to bigint, due to the large size of breaches

Some breaches are in the millions or even billions, so to make sure we can capture the whole value, it should be a big int instead of an integer datatype
